distributions/JavaScript/compile_to_js.sh

```diff
@@ -21,8 +21,457 @@ ls .
 emcmake cmake ..
 emmake make
 
+funs='['	
```

Comment:
Do we have a script which generates list of functions, as below ?
If yes, add a comment line mentioning how to do it

Reply:
It is described in JS_INSTALL.txt file

Reply:
ok